app/views/group_entities/index.html.erb

```diff
@@ -0,0 +1,14 @@
+<p style="color: green"><%= notice %></p>
+
+<h1>Group entities</h1>
+
+<div id="group_entities">
+  <% @group_entities.each do |group_entity| %>
+    <%= render group_entity %>
+    <p>
+      <%= link_to "Show this group entity", group_entity %>
+    </p>
+  <% end %>
+</div>
```

Comment:
- [ ] It would be better if we try to match the design. Below is a screenshot of the required design and our desing implementation so we can compare and fix accordingly
![Screenshot from 2023-07-04 22-53-57](https://github.com/calebchris000/Budget-App/assets/46388113/4b8f0f53-52cc-42ea-8eea-9857d071ed13)
![Screenshot from 2023-07-04 22-53-39](https://github.com/calebchris000/Budget-App/assets/46388113/9e0dc07f-176f-4151-a315-763086bd9e2b)
As we can see, the button should be fix at the buttom and the length of the content should be increase to match the design